app/src/main/java/com/practicum/playlistmaker/search/ui/TrackAdapter.kt

```diff
@@ -20,6 +21,7 @@ import kotlinx.coroutines.launch
 class TrackAdapter(
     private val context: Context,
     private var tracks: ArrayList<Track>,
```

Comment:
```suggestion
    private var tracks: List<Track>,
```

Нам не важна конкретная реализация списка